mobile.html

```diff
@@ -29,13 +29,13 @@
                 <div class="group-4">
```

Comment:
Kindly add your name to your title page:
![Screen Shot 2021-09-28 at 9 43 24](https://user-images.githubusercontent.com/22258274/135110013-23d8ffed-5d8c-4a1f-ac16-f352f31731e8.png)

A few examples to have a nice title:
- Akinlade Temitope's portifolio
- Akinlade Temitope full-stack webdeveloper